support-frontend/assets/pages/paper-subscription-checkout/components/summary.jsx

```diff
@@ -0,0 +1,230 @@
+// @flow
```

Comment:
Would be good to move this component to `support-frontend/assets/components/subscriptionCheckouts`, assuming it can be reused in the other checkouts

Reply:
I have just done so - it needed a few tweaks to work with the digital version. The initial idea was to roll it out to one product at a time.